modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -18,6 +18,11 @@ import * as Operations from './undo/Operations';
 import { Index, Locks, UndoBookmark, UndoLevel, UndoManager } from './undo/UndoManagerTypes';
 import { addVisualInternal } from './view/VisualAidsImpl';
 
+interface ExtraAPI {
```

Comment:
Could this share the type with Extra in UndoManager to have all 4 variants?

Reply:
I believe I had some issues importing it here, for some reason. I'll give it another try.